config/locales/jobseekers.yml

```
@@ -171,6 +171,28 @@ en:
           optional: You can select ‘prefer not to say’ if you would rather not answer any question.
           step_title: Equal opportunities
           title: Equal opportunities — Application
+        catholic_following_religion:
+          heading: Religious information
+          title: Religious information — Application
+          is_a_catholic_school: "%{name} is a Catholic school."
+          preference_to_catholics: Schools, academies and colleges of a religious character can give preference to Catholic applicants when recruiting for teaching roles.
+          preference_for_support_staff: They can also give preference to Catholic applicants for support staff roles where there is a genuine occupational requirement.
+          non_catholics_apply: Non-Catholics are still encouraged to apply.
```

```suggestion
          non_catholics_apply: Applicants who do not follow a faith or religion are still encouraged to apply.
```
@starswan can we tweak this please? :) (didnt want to change myself in case I mess up any code!!)